package chap10_exception;
// 2022.08.18.(목)
// 숫자 2개를 받아와서 나누는 클래스
public class MethodThrow {
	// 필드
	private int x,y;
	
	// 생성자
	public MethodThrow() {
		x = 1;
		y = 1;
	}
	
	public MethodThrow(int x, int y) {
		this.x = x;
		this.y = y;
	}
	
	// 상위 메소드 (객체를 통해 메소드를 사용할 때 try/catch 사용)
	// 상위 메소드에게 예외처리를 요청하는 throw
	public int division() throws Exception{
		return x/y;
	}
	
}
